app/views/bulk_update_mailer/update_status.html.erb

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h2>Bulk update status report for batch initiated on <%= @time %></h2>
+    <% if @resource_id.present? %> 
+      <p>
+        Bulk update failed due to invalid change set on Resource with id <%= @resource_id %>
```

Comment:
Since this'll be sent to users, maybe something like `due to invalid parameters on Resource with ID <%= @resource_id %>. If you're unsure of what to do with this error please contact DLS in #digital-library or via digital-library@princeton.libanswers.com.` ?

Maybe even provide a link to the resource?